### PR TITLE
Add vcpkg-robotology-yarp-only.zip package conaining only YARP dependencies

### PR DESCRIPTION
**Pull request by drdanz**

### CODE REVIEW

**Comment by traversaro** [comment]
Recommended way to view the PR: https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/pull/28/files?w=1 .

**Comment by traversaro** [comment]
I am not sure what happened, but apparently only ` build-yarp-deps-only` are not running.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -41,20 +41,20 @@ jobs:
     - name: Install vcpkg ports
       shell: bash
       run: |
-        # TinyXML is not installed as a workaround for https://github.com/robotology/ycm/pull/296
-        C:/robotology/vcpkg/vcpkg.exe --overlay-ports=C:/robotology/robotology-vcpkg-ports --overlay-ports=${GITHUB_WORKSPACE}/custom-ports install --triplet x64-windows ace asio boost-asio boost-bind boost-process boost-dll boost-filesystem boost-system freeglut gsl eigen3 esdcan-binary glew glfw3 ode openssl libxml2 libjpeg-turbo eigen3 opencv portaudio matio sdl1 sdl2 qt5-base[latest] ipopt-binary qt5-declarative qt5-multimedia qt5-quickcontrols qt5-quickcontrols2 sqlite3[core,tool]
+        C:/robotology/vcpkg/vcpkg.exe --overlay-ports=C:/robotology/robotology-vcpkg-ports --overlay-ports=${GITHUB_WORKSPACE}/custom-ports install --triplet x64-windows ace eigen3 gsl ipopt-binary libjpeg-turbo opencv portaudio sdl1 sdl2 qt5-base[latest] qt5-declarative qt5-multimedia qt5-quickcontrols qt5-quickcontrols2 sqlite3[core,tool] tinyxml
```
Comment (traversaro) [inline .github/workflows/ci.yml:44]: I think that `ipopt-binary` probably can be moved to the general archive, I don't think it  is used in YARP.

**Review by traversaro — APPROVED** [review]
Some really minor comments.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (traversaro) [reply .github/workflows/ci.yml:44]: Friendly ping @drdanz , if we prefer we can go forward even by keeping ipopt-binary in the YARP dependencies archive.

Comment (drdanz) [reply .github/workflows/ci.yml:44]: Sorry, I missed this... You are right, YARP does not use ipopt, I'll move it to the main package

**Comment by drdanz** [comment]
This is now compiling, but it is based on #35, and must be rebased before merging

**Comment by traversaro** [comment]
Thanks @drdanz ! I merged https://github.com/robotology/robotology-superbuild-dependencies-vcpkg/pull/35, so let me know if you want to rebase or I can just squash and merge!

**Comment by drdanz** [comment]
@traversaro done.

**Pull request merged**